code/datums/keybinding/human.dm

```diff
@@ -174,6 +174,24 @@
 	human_user.pickup_recent()
 	return TRUE
 
+/datum/keybinding/human/show_held_item
+	hotkey_keys = list()
+	classic_keys = list()
+	name = "show_held_item"
+	full_name = "Show Held Item"
+	keybind_signal = COMSIG_KB_HUMAN_SHOW_HELD_ITEM
+
+/datum/keybinding/human/pick_up/down(client/user)
```

Comment:
Harrryyyyyyyyyyy... You left the wrong path when copypasting

Reply:
i still think it should compile error

Reply:
well multiple overrides is valid DM and we got a few